rmgpy/tools/data/various_kinetics/chem_annotated.inp

```diff
@@ -172,21 +172,6 @@ HO2(5)+CH3(13)=O2(6)+CH4(15)                        1.000e+00 0.000     0.000
     PLOG/ 100.000   1.500e+11 0.000     1.987    /
     PLOG/ 650.000   2.300e+11 0.200     4.968    /
     PLOG/ 2000.000  3.700e+07 1.340     2.186    /
-DUPLICATE
```

Comment:
Can you elaborate on why this is needed? I worry it's indicative of a bigger problem, as I fear we have a couple of seed mechanisms that use this horrible sum-of-plog format. (mabye to be addressed in a different PR to this, but I don't want it to be forgotten)

Reply:
I will preface this by saying I think the last time the canteramodel tool in rmg was updated was when connie did the uncertainty analysis paper.  So it is outdated, I think before cantera would just ignore bad reactions. 
This is a little complicated. The error in 2.6 is: 
```
CanteraError: 
*******************************************************************************
InputFileError thrown by Kinetics::checkDuplicates:
Error on line 210 of [/Users/blais.ch/Documents/_01_code/RMG_env_1/testing_folder/2022_05_09_cantera_v2_5/chemkin_error.yaml](https://file+.vscode-resource.vscode-cdn.net/Users/blais.ch/Documents/_01_code/RMG_env_1/testing_folder/2022_05_09_cantera_v2_5/chemkin_error.yaml):
No duplicate found for declared duplicate reaction number 7 (CH3(13) + HO2(5) <=> CH4(15) + O2(6))
|  Line |
|   205 |   rate-constants:
|   206 |   - {P: 0.1 atm, A: 8.8e+16, b: -1.05, Ea: 6.461}
|   207 |   - {P: 1.0 atm, A: 8.0e+21, b: -2.39, Ea: 11.18}
|   208 |   - {P: 10.0 atm, A: 3.3e+24, b: -3.04, Ea: 15.61}
|   209 |   note: 'Reaction index: Chemkin #7; RMG #4'
>   210 > - equation: HO2(5) + CH3(13) <=> O2(6) + CH4(15)  # Reaction 8
            ^
|   211 |   duplicate: true
|   212 |   type: pressure-dependent-Arrhenius
|   213 |   rate-constants:
*******************************************************************************
```
I believe this is a bug that is addressed in later versions, but since it is for v3.0, which hasn't been released, I can't find release notes about it. Version 3.0 it accepts the duplicate, but gives the following error: 

```
Invalid rate coefficient for reaction 'CH3(13) + HO2(5) <=> CH4(15) + O2(6)'
at P = 2.0265e+08, T = 200.0
at P = 2.0265e+08, T = 500.0
at P = 2.0265e+08, T = 1000.0
at P = 2.0265e+08, T = 2000.0
at P = 2.0265e+08, T = 5000.0
at P = 2.0265e+08, T = 10000.0
```
This, I think, is expected behavior, as shown in the [plog.cpp program](https://github.com/Cantera/cantera/blob/e82f932119705596410719718e31a4ee63b2e48e/src/kinetics/PlogRate.cpp#L128). It is indicative of a negative rate. the A-factor at 2000 atm (2.0265e+08, Pa) is 0 so that's probably why. if 

both 3.0 and 2.6 function properly if I remove the rate at 2000 atm for reaction 9. further, version 2.6 gives me an "unmarked duplicate reaction" error if I fail to mark both reactions as duplicates, so I think the first error is due to cantera ignoring reaction 9 since it's invalid, and throwing an error for reaction 8, since it no longer has a duplicate. 

What I will do is remove this part of Reaction 9: 
```
PLOG/ 2000.000  0.000e+00 0.000     0.000    /
```
If I remove that both 2.6 and 3.0 can parse the mechanism and reload a `ct.Solution` object no problem. Other people are free to create an issue from  this, I am unsure if rmg has a validation check for library or generated plog reaction rates being negative at certain pressure/temperature combos.